pkg/template/app/app.go

```diff
@@ -57,6 +57,7 @@ func NewAppCR(config Config) ([]byte, error) {
 	if config.InCluster {
 		crNamespace = config.Namespace
 		appLabels[label.AppOperatorVersion] = "0.0.0"
+		appLabels[label.ManagedBy] = "flux"
```

Comment:
I think we only want this for Flux-managed clusters, right @MarcelMue? Maybe we should add a flag like "--cluster-app-extra-labels` so arbitrary labels could be passed in and it would be up to the user to set this.

Reply:
I think I added this as a recommendation from some slack thread, not sure if it's necessary or not.